packages/augur-sdk/src/state/getter/Universe.ts

```diff
@@ -36,6 +58,65 @@ export class Universe {
       fees: fees.toString(),
     };
   }
+
+  @Getter('getForkMigrationTotalsParams')
+  static async getForkMigrationTotals(
+    augur: Augur,
+    db: DB,
+    params: t.TypeOf<typeof Universe.getForkMigrationTotalsParams>
+  ): Promise<MigrationTotals | NonForkingMigrationTotals> {
+    const universeForkedLogs = await db.findUniverseForkedLogs({
+      selector: {
+        universe: params.universe,
+      },
+    });
+
+    if (universeForkedLogs.length === 0) {
+      return {};
+    }
+
+    const universe = universeForkedLogs[0];
+
+    const forkingMarket = (await db.findMarkets({
+      selector: {
+        market: universe.forkingMarket,
+      },
+    }))[0];
+    const marketTypeName = marketTypeToName(forkingMarket.marketType);
+    const numTicks = Number(forkingMarket.numTicks).toString(10);
+    const minPrice = Number(forkingMarket.prices[0]).toString(10);
+    const maxPrice = Number(forkingMarket.prices[forkingMarket.prices.length - 1]).toString(10);
+
+    const children = await db.findUniverseCreatedLogs({
+      selector: {
+        parentUniverse: params.universe,
+      },
+    });
+
+    const outcomes: Outcome[] = await Promise.all(children.map(async (child): Promise<Outcome> => {
+      const payoutNumerators = child.payoutNumerators.map((hex) => Number(hex).toString(10));
+
+      // TODO min and max prices probably need to be translated into display prices
+      const outcome = calculatePayoutNumeratorsValue(maxPrice, minPrice, numTicks, marketTypeName, payoutNumerators);
```

Comment:
there will be a new parameter to `calculatePayoutNumeratorsValue` called isInvalid